### DISCUSSION TITLE
Events V2

### DISCUSSION THREAD

**Discussion by djong1** [djong1/gp3s#35] [Ideas]
Based on the community request we are currently transforming the event module:

- one session can be attached to multiple events
- once a community member joins an event, his posted session will be automatically attached to the event, no manual selection or intervention from the user needed anymore
- events can run at world, country and spot level. previous its was spot based.

As a next step we would like to gather more needs from the community to create an even better event module. So tell us what you would like to be added to the event module to make it better to run your own event or to host existing events. We would like to use this discussion to gather requirements.

**Comment by Logiqx** [comment]
Good to see a plan for events running at country level.

Sessions are periodically posted to the British Speed Challenge from other countries, when the event rules state "British Waters".

**Comment by djong1** [comment]
implemented
